inbm/dispatcher-agent/tests/unit/source/test_ubuntu_source_cmd.py

```diff
@@ -251,7 +282,7 @@ def test_successfully_remove_gpg_key_and_source_list(
     @patch("dispatcher.source.ubuntu_source_manager.remove_gpg_key_if_exists")
     def test_raises_when_space_check_fails(self, mock_remove_gpg_key):
         parameters = ApplicationRemoveSourceParameters(
-            gpg_key_name="example_source.gpg", file_name="../example_source.list"
+            gpg_key_name="example_source.gpg", source_list_file_name="../example_source.list"
         )
         command = UbuntuApplicationSourceManager()
         with pytest.raises(SourceError) as ex:
```

Comment:
**Enhancement**
**Suggestion:** Use a more descriptive exception message that includes the file name when raising a SourceError due to space check failure.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        with pytest.raises(SourceError) as ex:
            command.remove(parameters)
        expected_message = f"Invalid file name: {parameters.source_list_file_name}"
        assert str(ex.value) == expected_message
```